src/rdkafka.c

```diff
@@ -889,6 +889,39 @@ rd_kafka_resp_err_t rd_kafka_test_fatal_error(rd_kafka_t *rk,
 }
 
 
+int rd_kafka_test_idemp_set_pid(rd_kafka_t *rk, int64_t id, int16_t epoch) {
+        rd_kafka_pid_t pid = {id, epoch};
+
+        if (!rd_kafka_is_idempotent(rk) || !rd_kafka_pid_valid(pid)) {
+                return 1;
```

Comment:
Split this into two if-statements:
 * for !is_idempotent return `RD_KAFKA_RESP_ERR__NOT_CONFIGURED`
 * for !pid_valid return `RD_KAFKA_RESP_ERR__INVALID_ARG`